examples/connext_dds/remote_procedure_call/c++11/USER_QOS_PROFILES.xml

```diff
@@ -4,147 +4,33 @@
+             the DataWriters and DataReaders created by a Requester or Replier
+        -->
+        <qos_profile name="RequesterExampleProfile" base_name="BuiltinQosLib::Pattern.RPC">
+            <!--
             <datareader_qos>
-                <!-- Strict reliable -->
-                <reliability>
-                    <kind>RELIABLE_RELIABILITY_QOS</kind>
-                    <max_blocking_time>
-                        <sec>10</sec>
-                        <nanosec>0</nanosec>
-                    </max_blocking_time>
-                </reliability>
-                
-                <history>
-                    <kind>KEEP_ALL_HISTORY_QOS</kind>
-                </history>
-                
-                <!-- These are typical protocol parameters for a reliable
-                     DataReader -->
-                <protocol>
-                    <rtps_reliable_reader>
-                        <max_heartbeat_response_delay>
-                            <sec>0</sec>
-                            <nanosec>0</nanosec>
-                        </max_heartbeat_response_delay>
-                        <min_heartbeat_response_delay>
-                            <sec>0</sec>
-                            <nanosec>0</nanosec>
-                        </min_heartbeat_response_delay>
-                    </rtps_reliable_reader>
-                </protocol>
-                
+                ...
             </datareader_qos>
-            
-        </qos_profile>
-    
-        <!-- This is the profile used by the Requester. 
-             It inherits from "default", defined above, 
-             and overrides some QoS -->
-        <qos_profile name="RequesterExampleProfile" 
-                     base_name="default">
-
-            <!-- QoS for the data writer that sends requests -->
             <datawriter_qos>
-                <durability>
-                    <kind>TRANSIENT_LOCAL_DURABILITY_QOS</kind>
-                </durability>
+                ...
             </datawriter_qos>
-
-            <!-- QoS for the data reader that receives replies -->
-            <datareader_qos>
-                <durability>
-                    <kind>VOLATILE_DURABILITY_QOS</kind>
-                </durability>
-            </datareader_qos>
+            -->
         </qos_profile>
 
-        <!-- This is the profile used by the Replier. 
-             It inherits from "default", defined above, 
-             and overrides some QoS -->
-        <qos_profile name="ReplierExampleProfile"
-                     base_name="default">
-        
-            <!-- QoS for the data writer that sends replies -->
-            <datawriter_qos>
-                <durability>
-                    <kind>VOLATILE_DURABILITY_QOS</kind>
-                </durability>
-            </datawriter_qos>
+        <qos_profile name="ReplierExampleProfile" base_name="BuiltinQosLib::Pattern.RPC">
 
-            <!-- QoS for the data reader that receives requests -->
```

Comment:
Why are you removing the text but keep the comment? 
I think that we should keep a comment with the meaning or remove the whole comment.